Review.md

```diff
@@ -1,12 +1,20 @@
 # Questions - Self Study
-
-You can exercise your Google-Fu for this and any other _Sprint Challenge_ in the future.
-
 1.  Explain the difference between `RDBMS` and `SQL`.
+        RDBMS is the system and SQL is the language we use to interact with it
+
 1.  Why do tables need a `primary key`?
-1.  What is the name given to a table column that references the primary key
-    on another table.
+        it allows each row in a table to have a unique number to reference to. no two rows can have the same primary key.
+
+1.  What is the name given to a table column that references the primary key on another table.
+        this is called a foreign key
+
 1.  What do we need in order to have a _many to many_ relationship between two
-    tables.
+        tables. this occurs when you have data in a table that is assocciated with data in another table
```

Comment:
Overall true but incomplete. "data in a table associated with data in another table" is a bit to ambiguous. One entry in one table could have a reference to one entry in another table (one to one). Many entries in one table could have reference to one entry in another table (one-to-many) and many entries in one table could have reference to many distinct entries in another table (many-to-many). Overall the "textbook" answer for the requirement of a many to many relationship is "the two tables in question share a one-to-many relationship with a third table.